modules/python-modules/syslogng/modules/kubernetes/scl/kubernetes.conf

```
@@ -91,7 +92,7 @@ block parser kubernetes-metadata-parser (prefix() template("$FILE_NAME")) {
      };
 };
 
-block parser kubernetes-file-parser(prefix('.k8s.') cluster-name('k8s')) {
+block parser kubernetes-file-parser(prefix('.k8s.') cluster-name('k8s') key-delimiter() {
```

A closing bracket is missing.
```suggestion
block parser kubernetes-file-parser(prefix('.k8s.') cluster-name('k8s') key-delimiter()) {
```